node/node_test.go

```diff
@@ -97,8 +99,10 @@ func TestAddPeers(t *testing.T) {
 		t.Fatalf("newhost failure: %v", err)
 	}
 	consensus := consensus.New(host, "0", []p2p.Peer{leader, validator}, leader)
+	dRand := drand.New(host, "0", []p2p.Peer{leader, validator}, leader, nil)
```

Comment:
there are many logic in the main right now. eventually we'll clean up the main file like this https://github.com/ethereum/go-ethereum/blob/master/cmd/geth/main.go#L268

i don't know a right place but probably put this one into node package temporarily.

Reply:
ok, will do.